packages/x-client/src/utils/crypto/crypto.test.ts

```diff
@@ -36,3 +38,17 @@ describe('signMessage()', () => {
     });
   });
 });
+
+describe('signRegisterEthAddress()', () => {
+  test('Correctly signs message', async () => {
+    const signer = new Wallet('5c7b4b5cad9a3fc7b1ba235a49cd74e615488a18b0d6a531739fd1062935104d');
```

Comment:
small thing: ethers allows you to create random wallets for testing https://docs.ethers.org/v4/api-wallet.html#creating-instances

Reply:
Oh nice, I just copied from the other tests but I would like to use this in future